recipes/pandoc/notes.md

```diff
@@ -0,0 +1,3 @@
+* The Linux package doesn't work on systems older than CentOS 6
+* The OS X package works on 10.7+
+* The Windows package works on 32 and 64 bit archs
```

Comment:
Does this information get included on the Anaconda.org package description,  If not could it be added to the summary section of the meta.yaml file?